modules/serverless-sinkbinding-kn.adoc

```diff
@@ -12,8 +12,37 @@ This guide describes the steps required to create, manage, and delete a SinkBind
 * You have Knative Serving and Eventing installed.
 * You have the `kn` CLI installed.
 
+[NOTE]
+====
+The following procedure requires you to create YAML files.
+
+If you change the names of the YAML files from those used in the examples, you must ensure that you also update the corresponding CLI commands.
+====
+
 .Procedure
 
+. To use a SinkBinding, cluster administrators must label the namespace configured in the SinkBinding with `bindings.knative.dev/include:"true"`:
++
+
+[source,terminal]
+----
+$ oc label namespace <namespace> bindings.knative.dev/include=true
+----
+
+. Users must manually add a `bindings.knative.dev/include=true` label to resources.
```

Comment:
I don't think this is a separate step. I think that this information should be pointed out twice, once in the previous step about adding the namespace label and then later in the procedure when you create the YAML file.

Reply:
OK, one last stab at it :smile: 

So, what I have done now is
- include the note for admins in the assembly, and in the prerequisites for each module (labeling namespace)
- include the note for users about labeling resources in the section where they create the cronjob YAML

Hopefully this looks better :slightly_smiling_face: